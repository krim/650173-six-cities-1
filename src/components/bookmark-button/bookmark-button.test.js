import React from 'react';
import renderer from 'react-test-renderer';

import BookmarkButton from './bookmark-button';

describe(`BookmarkButton`, () => {
  it(`renders component correctly`, () => {
    const tree = renderer.
      create(
          <BookmarkButton
            className={`property`}
            width={`10`}
            height={`20`}
            onBookmarkClick={jest.fn()}
            isFavorite={true}
          />
      ).
      toJSON();

    expect(tree).toMatchSnapshot();
  });
});
