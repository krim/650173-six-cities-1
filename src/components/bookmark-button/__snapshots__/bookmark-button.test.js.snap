// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkButton renders component correctly 1`] = `
<button
  className="property__bookmark-button button place-card__bookmark-button--active"
  onClick={[MockFunction]}
  type="button"
>
  <svg
    className="place-card__bookmark-icon"
    height="20"
    width="10"
  >
    <use
      xlinkHref="#icon-bookmark"
    />
  </svg>
  <span
    className="visually-hidden"
  >
    To bookmarks
  </span>
</button>
`;
